webapplication/sip/settings.py

```diff
@@ -251,4 +251,6 @@
 SITE_ID = 1
 
 DEFAULT_SYSTEM_NOTIFICATION_EMAIL = ""
+DEFAULT_USER_TRANSACTION_STATUS = "Something went wrong, please contact us"
```

Comment:
This is a bit strange for a default transaction status. The name is misleading.

Reply:
fixed